src/migrate/index.js

```diff
@@ -121,21 +139,72 @@ export default class Migrator {
     });
   }
 
+  _createMigrationLockTable(tableName) {
+    return this.knex.schema.createTable(tableName, function(t) {
+      t.boolean('is_locked');
+    });
+  }
+
+  _getLockTableName(tableName) {
+    return tableName + '_lock';
+  }
+
+  _isLocked() {
+    var tableName = this._getLockTableName(this.config.tableName);
+    return this.knex(tableName)
+      .select('*')
+      .then(function(data) {
+        if (data[0].is_locked) {
```

Comment:
`is_locked` is already boolean, could you just return `.then((data) => data[0].is_locked)`